include/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.hpp

```diff
@@ -75,6 +81,17 @@ class Segment
         Real computeDeltaV(const Real& aSpecificImpulse) const;
         Mass computeDeltaMass() const;
 
+        Map<Shared<Dynamics>, MatrixXd> getDynamicsContributions(const Shared<const Frame>& aFrameSPtr) const;
+        MatrixXd getDynamicsContribution(
```

Comment:
I think we're missing docs for these functions. Don't know how important it is.
But if we add them we could state that "at the moment, the dynamics are recomputed"